loops-arrays-objects.js

```diff
@@ -9,15 +9,26 @@ const scores = [
     {name: 'Rianne', score: 66, grade: null}
 ];
 
-// Verwachtte uitkomsten:
-// 83
-// 77
-// 92
-// 66
-// ==========================================
-
-
-
+for (let i = 0; i < scores.length; i++) {
+
+    if (scores[i].score < 60) {
+        scores[i].grade ='F'
+    }
+    else if (scores[i].score < 70) {
+        scores[i].grade ='D'
+    }
+    else if (scores[i].score < 80) {
+        scores[i].grade ='C'
+    }
+    else if (scores[i].score < 90) {
+        scores[i].grade ='B'
+    }
+    else {
+        scores[i].grade ='A'
+    }
+
+}
+console.log(scores)
 
```

Comment:
Je hebt hier stappen A en B overgeslagen en je bent gelijk naar D gegaan. Jouw uitwerking is exact hetzelfde als de uitwerkingen zelf. Code is daardoor goed!